client/landing/stepper/declarative-flow/internals/index.tsx

```diff
@@ -59,7 +59,7 @@ export const FlowRenderer: React.FC< { flow: Flow } > = ( { flow } ) => {
 				} ),
 				{}
 			),
-		[]
+		[ stepPaths ]
```

Comment:
This fixes the issue where Woo flow displayed a blank page on the first page load.

Reply:
What happens if we use `flowStep` as a dependency instead? I can see that `flowStep` is an actual dependency of the hook and we aren't providing it.

Reply:
Good idea! I used the stepPath because it contained the paths as strings and thought that `flowSteps` might cause unnecessary re-renders since it contains a list of objects.